pkg/ccloudv2/networking.go

```diff
@@ -32,8 +32,13 @@ func (c *Client) DeleteNetwork(environment, id string) error {
 	return errors.CatchCCloudV2Error(err, httpResp)
 }
 
-func (c *Client) UpdateNetwork(environment, id string, updateReq networkingv1.NetworkingV1NetworkUpdate) (networkingv1.NetworkingV1Network, error) {
-	resp, httpResp, err := c.NetworkingClient.NetworksNetworkingV1Api.UpdateNetworkingV1Network(c.networkingApiContext(), id).NetworkingV1NetworkUpdate(updateReq).Execute()
+func (c *Client) UpdateNetwork(environment, id string, networkingV1NetworkUpdate networkingv1.NetworkingV1NetworkUpdate) (networkingv1.NetworkingV1Network, error) {
+	resp, httpResp, err := c.NetworkingClient.NetworksNetworkingV1Api.UpdateNetworkingV1Network(c.networkingApiContext(), id).NetworkingV1NetworkUpdate(networkingV1NetworkUpdate).Execute()
+	return resp, errors.CatchCCloudV2Error(err, httpResp)
+}
+
+func (c *Client) CreateNetwork(networkingV1Network networkingv1.NetworkingV1Network) (networkingv1.NetworkingV1Network, error) {
```

Comment:
```suggestion
func (c *Client) CreateNetwork(network networkingv1.NetworkingV1Network) (networkingv1.NetworkingV1Network, error) {
```

Reply:
I can change this to `network` but I have the impression from this previous [PR](https://github.com/confluentinc/cli/pull/2218#discussion_r1303329873) that we want to match the parameter name with the setter function name. Or does that apply to only `environment`?

Reply:
We don't have any strict rules (feel free to ignore these suggestions) but in general, I just name these variables after the resource name, i.e. "environment" or "network" which usually matches the setter name. In this case the setter name is a bit redundant since it was auto-generated...

Reply:
yeah makes sense, i will update it